app/views/categories/_category.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= link_to category do %>
+  <div id="<%= dom_id category %>" class="flex justify-between items-center	bg-white p-5">
+    <p class="my-5 text-2xl">
+      <%= category.icon %>
+    </p>
+    
+    <p class="my-5">
+      <%= category.name %>
+    </p>
+
+    <p class="font-bold">
+      <%= number_to_currency(category.expenses.sum(:amount)) %>
+    </p>
+
+    <%# if action_name != "show" %>
+      <%#= link_to "Show this category", category, class: "rounded-lg py-3 px-5 bg-gray-100 inline-block font-medium" %>
+      <%#= link_to 'Edit this category', edit_category_path(category), class: "rounded-lg py-3 ml-2 px-5 bg-gray-100 inline-block font-medium" %>
+      <%# <hr class="mt-6"> %>
+    <%# end %>
```

Comment:
- [ ] It's a good convention to not have commented codes in the codebase as they make our codebase dirty and not professional. Kindly remove all commented codes from your codebase so your code looks clean and professional 🙃 😉

Reply:
I forgot about this comment. Thank you!